variables.tf

```diff
@@ -22,30 +22,55 @@
 # Configure the IBM Cloud provider
 ################################################################
 variable "ibmcloud_api_key" {
+    type        = string
     description = "IBM Cloud API key associated with user's identity"
     default = "<key>"
+
+    validation{
+        condition = var.ibmcloud_api_key != "" && lower(var.ibmcloud_api_key) != lower("<key>")
```

Comment:
`lower("<key>")` -> `"<key>"` since it is already lower as defined in string,

Reply:
Checked in latest code change fix by removing lower() and word corrected.